packages/backend-core/src/users/users.ts

```diff
@@ -276,7 +277,7 @@ export const paginatedUsers = async ({
     const response = await db.allDocs(getGlobalUserParams(null, opts))
     userList = response.rows.map((row: any) => row.doc)
   }
-  return pagination(userList, pageLimit, {
+  return pagination(userList, pageSize, {
```

Comment:
We were previously passing in the same number here as the number of docs we were fetching. Inside here we check if the data length is longer than the desired page limit, but since this was the same number that was always false. We should have been passing in the actual page size, rather than page size + 1.